examples/elbv2/lbtargetgroupattachment.yaml

```diff
@@ -29,3 +29,102 @@ spec:
       matchLabels:
         testing.upbound.io/example-name: elbv2
     targetId: 172.16.10.25
+---
+apiVersion: ec2.aws.upbound.io/v1beta1
```

Comment:
Could you please add the `meta.upbound.io/example-id: elbv2/v1beta1/lbtargetgroupattachment` annotation to the dependencies also?